receiver/kubeletstatsreceiver/e2e_test.go

```diff
@@ -27,6 +27,13 @@ import (
 
 const testKubeConfig = "/tmp/kube-config-otelcol-e2e-testing"
 
+// The test requires a prebuilt otelcontribcol image uploaded to a kind k8s cluster defined in
+// `/tmp/kube-config-otelcol-e2e-testing`. Run the following command prior to running the test locally:
+//
+//	kind create cluster --kubeconfig=/tmp/kube-config-otelcol-e2e-testing --config=./.github/workflows/configs/e2e-kind-config.yaml (from the top level dir of the repository).
```

Comment:
```suggestion
//	kind create cluster --kubeconfig=/tmp/kube-config-otelcol-e2e-testing --config=./.github/workflows/configs/e2e-kind-config.yaml # Run from the top level dir of the repository
```
Nit: If the user wants to just copy/paste the commands it will make it a bit easier to just comment the last part rather than using parentheses 👍